src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -1150,8 +1150,8 @@ public void BuildApplicationWithJavaSource (bool isRelease, bool expectedResult)
 						var runtime = runtimeInfo.FirstOrDefault (x => x.Abi == abi && x.Runtime == expectedRuntime);
 						Assert.IsNotNull (runtime, "Could not find the expected runtime.");
 						var inApk = ZipHelper.ReadFileFromZip (apk, String.Format ("lib/{0}/{1}", abi, runtime.Name));
-						var inApkRuntime = runtimeInfo.FirstOrDefault (x => x.Abi == abi && x.Size == inApk.Length);
-						Assert.IsNotNull (inApkRuntime, "Could not find the actual runtime used.");
+						var inApkRuntime = runtimeInfo.FirstOrDefault (x => x.Abi == abi && x.Runtime == expectedRuntime);
+						Assert.AreNotEqual (default (ProjectBuilder.RuntimeInfo), inApkRuntime, "Could not find the actual runtime used.");
```

Comment:
yup null check on a struct ... doh.  bad idea, it was probably a class at some point and I changed it.